modules/ipi-verifying-nodes-after-installation.adoc

```diff
@@ -0,0 +1,68 @@
+
+[role="_abstract"]
+//Short introductory paragraph that provides an overview of the module. The text that immediately follows the abstract tag is used for search metadata.
+Your OpenShift Container Platform installation completes when the following installation health checks are successful:
+
+* The provisioning host can access the OpenShift Web Console.
+
+* All master nodes are ready.
+
+* All cluster Operators are available.
+
+However, although your installation successfully completes, not all worker nodes in the cluster are instantly provisioned. Some worker nodes remain in a non-provisioned state, such as NotReady, Inspecting or Provisioned. 
+
+The cluster Operator nodes continuously attempt to provision worker nodes for a set period of time. The state of a worker node only changes to degraded when the cluster Operator node fails to provision the worker node within this time period. You can define this time period in the install-config.yaml(???) file.
+
+After your installation completes, you can continue to monitor the condition of the nodes in your cluster using the following steps. 
+
+.Prerequisites
+//A bulleted list of conditions that must be satisfied before the user starts following this module.
+* The installation program resolves successfully in the terminal.
+
+.Procedure
+//Start each step with an active verb. Use an unnumbered bullet (*) if the procedure includes only one step.
+. Show the status of all worker nodes:
++
+[source,terminal]
+----
+oc get nodes
+----
++
+.Example output
+[source,terminal]
+NAME                           STATUS   ROLES    AGE   VERSION
+example-compute1.example.com   Ready    worker   13m   v1.21.6+bb8d50a
+example-compute2.example.com   Ready    worker   13m   v1.21.6+bb8d50a
+example-compute4.example.com   Ready    worker   14m   v1.21.6+bb8d50a
+example-control1.example.com   Ready    master   52m   v1.21.6+bb8d50a
+example-control2.example.com   Ready    master   55m   v1.21.6+bb8d50a
+example-control3.example.com   Ready    master   55m   v1.21.6+bb8d50a
+
+. Show the phase of all worker machine nodes:
++
+----
+oc get machines -A
+----
++
+.Example output
+[source,terminal]
+NAMESPACE               NAME                           PHASE         TYPE   REGION   ZONE   AGE
+openshift-machine-api   example-zbbt6-master-0         Running                              95m
+openshift-machine-api   example-zbbt6-master-1         Running                              95m
+openshift-machine-api   example-zbbt6-master-2         Running                              95m
+openshift-machine-api   example-zbbt6-worker-0-25bhp   Running                              49m
+openshift-machine-api   example-zbbt6-worker-0-8b4c2   Running                              49m
+openshift-machine-api   example-zbbt6-worker-0-jkbqt   Provisioned                          49m
+openshift-machine-api   example-zbbt6-worker-0-qrl5b   Running                              49m
+
+[role="_additional-resources"]
```

Comment:
No xrefs in modules remember :) include this in the asembly, just after the module include reference.

Reply:
Fixed